ansible/scripts/install_ansible.sh

```diff
@@ -22,9 +22,19 @@ base_dir=$( cd "$( dirname "$bin" )" && pwd )
 
 set -e
 
+# get the release id
+if [ -f /etc/os-release ]; then
+  . /etc/os-release
+  ID=$ID
+fi
+
 # enable yum epel repo
-is_installed_epel_release="rpm -q --quiet epel-release"
-install_epel_release="sudo yum install -q -y epel-release"
+if [ $ID = "rhel" ]; then
+  is_installed_epel_release="rpm -q --quiet epel-release"
+else
+  install_epel_release="sudo yum install -q -y epel-release"
```

Comment:
Seems like `install_epel_release` will always be overridden immediately after the if/else. If so, why set it here?

Reply:
Thanks; that was a bug. I pushed a new commit that fixes that.